Breaking up envvars and config settings into separate settings pages (#1028)
minio/docs: source/reference/minio-server/settings/root-credentials.rst

.. _minio-server-envvar-root:

===============

> `root` Settings

Reviewer: Maybe this is a stray markdownism?
```suggestion
``root`` Settings
```

Alternatively, could just skip inline code format in this heading and go with `Root`.

Author: Yeah, going to omit the formatting. Monotext makes the headings and left nav look weird anyway.

===============

.. default-domain:: minio

.. contents:: Table of Contents
   :local:
   :depth: 2

> This page covers settings that control root access for the MinIO process. 

Author: Done.

You can define settings by defining 

- an *environment variable* prior to starting or restarting the MinIO Server.
- a *configuration setting* using :mc:`mc admin config set`.
- a *configuration setting* using the :ref:`MinIO Console's <minio-console-settings>` :guilabel:`Administrator > Settings` pages.
  
If you define both an environment variable and the similar configuration setting, MinIO uses the environment variable value.

> Some settings can only be defined by either an environment variable or by a configuration setting.

Reviewer: An idea for your consideration:
```suggestion
Some settings have only an environment variable or a configuration setting but not both.
```

Author: I like it.
